src/Toolkit/Toolkit.WinUI/UI/Controls/PopupViewer/PopupViewer.Theme.xaml

```diff
@@ -1,69 +1,202 @@
+                        <TextBlock Grid.Row="1" Grid.ColumnSpan="3" DataContext="{TemplateBinding Element}" Text="{Binding Description, Mode=OneWay}" Style="{StaticResource PopupViewerCaptionStyle}" Visibility="{Binding Description, Converter={StaticResource PopupViewerVisibilityConverter}}" />
+
+                        <StackPanel Grid.Row="2" Margin="0,0,0,20" DataContext="{TemplateBinding CurrentItem}" Grid.Column="1">
+                            <primitives:PopupMediaView PopupMedia="{Binding}" Margin="0,5" />
+                            <TextBlock Text="{Binding Title, Mode=OneWay}" Style="{StaticResource PopupViewerTitleStyle}" Visibility="{Binding Title, Converter={StaticResource PopupViewerVisibilityConverter}}" />
+                            <TextBlock Text="{Binding Caption, Mode=OneWay}" Style="{StaticResource PopupViewerCaptionStyle}" Visibility="{Binding Caption, Converter={StaticResource PopupViewerVisibilityConverter}}" />
+                        </StackPanel>
+                        <Button Grid.Column="0" Grid.Row="2"  Content="" Background="Transparent" Height="40" BorderThickness="0" FontFamily="Segoe MDL2 Assets" VerticalAlignment="Center" x:Name="PreviousButton" />
+                        <Button Grid.Column="2" Grid.Row="2"  Content="" Background="Transparent" Height="40" BorderThickness="0" FontFamily="Segoe MDL2 Assets" VerticalAlignment="Center" x:Name="NextButton" />
+                    </Grid>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style TargetType="controls:PopupViewer" >
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:PopupViewer">
+                    <Border Background="{TemplateBinding Background}"
+                   BorderBrush="{TemplateBinding BorderBrush}"
+                   BorderThickness="{TemplateBinding BorderThickness}"
+                   Padding="{TemplateBinding Padding}"
+                   Margin="{TemplateBinding Margin}"
+                   MinWidth="{TemplateBinding MinWidth}"
+                   MaxWidth="{TemplateBinding MaxWidth}">
+                        <Grid DataContext="{TemplateBinding Popup}">
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto"/>
+                                <RowDefinition Height="*"/>
+                            </Grid.RowDefinitions>
+                            <TextBlock Text="{Binding Title, Converter={StaticResource PopupViewerHtmlToPlainTextConverter}, Mode=OneWay}" Style="{StaticResource PopupViewerHeaderStyle}"
+                         Visibility="{Binding Title, Converter={StaticResource PopupViewerVisibilityConverter}}" />
+                            <ScrollViewer VerticalScrollBarVisibility="{TemplateBinding VerticalScrollBarVisibility}"  Grid.Row="1" x:Name="PopupContentScrollViewer">
```

Comment:
Is there a way to keep the scrollbar from overlapping the views?  Scrollbar currently renders on top of text and gallery buttons, obscuring them:

![2024-08-02_012239 Toolkit SampleApp WinUI](https://github.com/user-attachments/assets/90967559-24bb-4b07-8a10-2d081d8445ee)